### PR TITLE
Consolidate toward name SeaweedFS

### PR DESCRIPTION
**Pull request by stuartpb**
This updates README.md to consistently use the name "SeaweedFS", per #94.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 Seaweed File System
 =======
 
-[![Build Status](https://travis-ci.org/chrislusf/weed-fs.svg?branch=master)](https://travis-ci.org/chrislusf/weed-fs)
-[![GoDoc](https://godoc.org/github.com/chrislusf/weed-fs/go?status.svg)](https://godoc.org/github.com/chrislusf/weed-fs/go)
-[![RTD](https://readthedocs.org/projects/weed-fs/badge/?version=latest)](http://weed-fs.readthedocs.org/en/latest/)
+[![Build Status](https://travis-ci.org/chrislusf/seaweedfs.svg?branch=master)](https://travis-ci.org/chrislusf/seaweedfs)
+[![GoDoc](https://godoc.org/github.com/chrislusf/seaweedfs/go?status.svg)](https://godoc.org/github.com/chrislusf/seaweedfs/go)
+[![RTD](https://readthedocs.org/projects/seaweedfs/badge/?version=latest)](http://seaweedfs.readthedocs.org/en/latest/)
```
Comment (chrislusf) [inline README.md]: This link is not working: http://seaweedfs.readthedocs.org/en/latest/
Anyway to make the link work?

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ http://groups.google.com/group/weed-file-system Seaweed File System Discussion G
 * Automatic compaction to reclaimed disk spaces after deletion or update
 * Servers in the same cluster can have different disk spaces, different file systems, different OS
 * Adding/Removing servers do not cause any data re-balancing
-* Optional  [filer server](https://code.google.com/p/weed-fs/wiki/DirectoriesAndFiles) provides "normal" directories and files via http
+* Optional  [filer server](https://github.com/chrislusf/seaweedfs/wiki/Directories-and-Files) provides "normal" directories and files via http
```
Comment (chrislusf) [inline README.md]: seems the link not working. Is there any way to redirect? or just use this?
  https://github.com/chrislusf/weed-fs/wiki/Directories-and-Files

Comment (stuartpb) [inline README.md]: These links are written to direct to the wiki after renaming the repo to https://github.com/chrislusf/seaweedfs .

File: README.md (hunk 1 above)
Comment (stuartpb) [inline README.md]: I assumed the link would change if the repo were renamed on GitHub - I'm looking into how renames work on readthedocs.org now.

Comment (stuartpb) [inline README.md]: This rename looks like it might be tricky, so I've reverted this change for now.

Comment (stuartpb) [inline README.md]: I just realized the /docs directory is the same content as the old wiki on Google Code - it seems like it'd be simpler to just use the GitHub wiki.

**Comment by stuartpb** [comment]
I've rewritten the patch to only address the name in text. The link changes were premature, and can be handled in a separate patch later.

**Pull request merged**